kurtosis-module/impl/module.go

```diff
@@ -153,8 +154,8 @@ func (e Eth2KurtosisModule) Execute(enclaveCtx *enclaves.EnclaveContext, seriali
 	logrus.Info("Launching grafana...")
 	err = grafana.LaunchGrafana(
 		enclaveCtx,
-		grafanaDatasourceConfigTemplate,
-		grafanaDashboardsConfigTemplate,
+		string(grafanaDatasourceConfigTemplate),
```

Comment:
Slight preference for doing the string cast earlier, in a single spot, so we don't have to keep casting